### ISSUE TITLE
ctrl + c doesn't work in chat

### ISSUE THREAD

**Issue by SolatK** [NebulaSS13/Nebula#2076]
#### Description of issue
Cant copy text from game chat using ctrl c

#### Steps to reproduce
1) select some letters 
2) press ctrl + c
3) Did not work
#### Client version, Server revision & Game ID

Running revision:
2021-09-10
1f703d4f8dacfecea7f5f7922ba25ea695177159

#### Issue bingo

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/NebulaSS13/Nebula/issues)

**Comment by quardbreak** [comment]
I researched a bit and have some thoughts about this situation:

Calling `erase_all_macros()` on client will erase all keybinds for `macro "default"` in skin. I did this to check if something conflicting with combination from this issue. After erase, Ctrl+C combination start working again.

The theory is that SSinput can steal keypresses, one of which is Ctrl. Probably, other servers don't have such issue with copying because they use browser window to output messages. Output window and browser window can work differently in this case. If this true, only one way that can fix this, I think, is to switch output type to browser window.
